Remotion/Web/Development.WebTesting.IntegrationTests/FileDownloadTest.cs

```diff
@@ -145,33 +128,7 @@ public void TestDownloadOpensInNewWindow_DownloadTimeoutExceeded_WithUnknownFile
             button.Click();
           });
 
-      if (Helper.BrowserConfiguration.IsInternetExplorer())
-      {
-        var downloadDirectory = ((InternetExplorerDownloadHelper) Helper.BrowserConfiguration.DownloadHelper).DownloadDirectory;
-        var filesInDownloadDirectoryBeforeDownload = Directory.GetFiles (downloadDirectory);
-
-        startDownloadLambda();
-
-        Assert.That (
-          () => Helper.BrowserConfiguration.DownloadHelper.HandleDownloadWithDetectedFileName (downloadStartedTimout, downloadUpdatedTimeout),
-          Throws.InstanceOf<DownloadResultNotFoundException>()
-              .With.Message.StartsWith (
-                string.Format (
-                  "The download result file did not get updated for longer than the downloadUpdatedTimeout of '{0}'. The download appears to have failed.",
-                  downloadUpdatedTimeout)));
-
-        //The browser continues writing in the download file, therefore we have to wait a little bit, so that it does not interfere with the next test.
-        Thread.Sleep (TimeSpan.FromSeconds (3));
-
-        //InternetExplorer sometimes opens a download information bar to inform us, that the downloaded file was deleted. 
-        //We close the bar (by sending the keys "alt + q") as it potentially interferes with future tests.
-        SendKeys.SendWait ("%q");
-
-        //InternetExplorer works in the default download directory of the user. If cleanUpUnmatchedDownloadedFiles is set to false in config, the unmatched file is not deleted by the WebTestFramework. 
-        //Therefore, we have to delete these files manually.
-        CleanupPartialSampleFiles (downloadDirectory, filesInDownloadDirectoryBeforeDownload);
-      }
-      else if (Helper.BrowserConfiguration.IsChrome())
+      if (Helper.BrowserConfiguration.IsChrome())

```

Comment:
why do we have this only in chrome but not in edge? (firefox is obviously different.